### PR TITLE
 Tachyon-318: update the web interface page

### PR DESCRIPTION
**Pull request by ShaoshanLiu**
all IP addresses are covered with red marks in order not to expose sensitive information

### CODE REVIEW

File: docs/Web-Interface.md [hunk 1]
```diff
@@ -4,23 +4,30 @@ title: Web Interface
...
-describe its homepage, browsing page, and file page.
+describe its details.
 
 The **home page** gives an overview of the system's status.
 
-![home](./img/screenshot_home.png)
+![home](./img/screenshot_overview.png)
 
+To check the list of all workers, click "Workers" button. In **workers page**, all nodes are listed,
+also the last heartbeat, node state, workers capacity, spaced used in bytes, as well as space
```
Comment (calvinjia) [inline docs/Web-Interface.md]: typo: space used in bytes

**Comment by calvinjia** [comment]
Thanks for the patch! Would it be possible to add a bit of info about the overview page since the display at the bottom is new (tiered storage info)?

**Comment by ShaoshanLiu** [comment]
Sure, let me add that info now.

**Comment by haoyuan** [comment]
Thanks!

**Pull request merged**